.github/workflows/pr_label_checks.yml

```diff
@@ -0,0 +1,20 @@
+name: PR Label Checks
+
+on:
+  pull_request:
+    types: [labeled, unlabeled]
```

Comment:
Q: Why run this on unlabeled PRs if the workflow checks for a label? Or does it not work that way?

Reply:
This ensures that if the check was previously failing (i.e. ran on a `labeled` action) that it runs again (and passes) when it's `unlabeled`

Reply:
Ooooh. OK.